spec/tests_acceptance/api_spec_acceptance.rb

```diff
@@ -45,136 +45,143 @@ def app
       get "/api/v1/projects/#{USERNAME}/#{PROJECT_NAME}"
       _(last_response.status).must_equal 202
 
-      5.times { sleep(1); print '.' }
+      40.times do
```

Comment:
Do we really need a 40 sec delay?!?

Reply:
Yes...Sometimes the worker may take more than 40 sec to run....